cucumber/authenticators_jwt/features/authn_jwt_configuration.feature

```diff
@@ -144,3 +144,255 @@ Feature: JWT Authenticator - Configuration Check
     """
     CONJ00086E Signing key URI configuration is invalid
     """
+
+  Scenario: provider-uri configured with correct value, jwks-uri configured with empty value, error
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "jwks-uri" variable to value " "
+    And I successfully set authn-jwt "provider-uri" variable to value "provider uri placeholder"
+    And I permit host "myapp" to "execute" it
```

Comment:
Also this step is redundant because you are failing before you validating the user